lib/definitions/xcode.d.ts

```diff
@@ -1,12 +1,12 @@
 declare module "xcode" {
-    interface FrameworkOptions {
+    interface Options {
         [key: string]: any;
 
         customFramework?: boolean;
-
         embed?: boolean;
+        relativePath?: string;
     }
-
+        
```

Comment:
Hmm, why does this empty space not trigger the linter? At any rate, please remove it.